can_tools/scrapers/__init__.py

```diff
@@ -193,7 +193,6 @@
 )
 from can_tools.scrapers.official.WY.WYVaccines import (
     WYCountyVaccinations,
-    WYStateVaccinations,
```

Comment:
this scraper keeps breaking, and since we don't currently, and have never used (afaik), state-sourced state-level data, I want to just remove this scraper